src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -3084,7 +3084,7 @@ public boolean isReflectivePreviewAPI(Element el) {
              !previewAPIs.previewAPI.isEmpty() ||
              !previewAPIs.reflectivePreviewAPI.isEmpty() ||
              !previewAPIs.declaredUsingPreviewFeature.isEmpty()) &&
-            !hasNoProviewAnnotation(el)) {
+            !hasNoPreviewAnnotation(el)) {
```

Comment:
I know this fix is just a typo fix, but that aside, the if-expression would be improved with more indentation to help show the binding of the parens, `||` and `&&. A default indentation of ` character is too small in this case, IMO.

Reply:
Don't bother: a subsequent merge with the mainline made that particular change irrelevant. Someone else found that issue and fixed it in parallel with my fix.

Reply:
Don't bother: a subsequent merge with the mainline (see JDK-8277300) made that particular change irrelevant and fixed that indentation in one go.